converters/toZigbee.js

```diff
@@ -683,33 +695,6 @@ const converters = {
             await entity.command('hvacThermostat', 'getRelayStatusLog', {}, getOptions(meta));
         },
     },
-    thermostat_weekly_schedule_rsp: {
```

Comment:
Responses weekly_schedule_rsp and relay_status_log_rsp in toZigbee list don't work so I removed them. Unfortunately my thermostat doesn't support relay_status_log so I was not able to test it but weekly_schedule fully tested.